.github/workflows/maven-test.yml

```diff
@@ -18,5 +18,15 @@ jobs:
         java-version: '11'
         distribution: 'temurin'
         cache: maven
-    - name: Run the Maven verify phase
-      run: mvn -B --update-snapshots verify
+    - name: Run Maven Build Step
+      run: mvn -B -U -DskipTests package
+    - name: Run Maven Test Step
```

Comment:
Why do we run maven twice, once without tests and once with tests?

Reply:
Does not work in one step - for publishing the report, this is the only way it works.